test-app/src/main/AndroidManifest.xml

```diff
@@ -47,6 +47,12 @@
         android:label="@string/title_trip_progress">
     </activity>
 
+    <activity
+        android:name=".MapboxManeuverActivity"
+        android:label="@string/title_maneuver"
+        android:screenOrientation="portrait">
```

Comment:
Could you unblock the rotation?

Reply:
I can but then I would have to handle the rotation in my example activity which we don't want to as discussed [here](https://github.com/mapbox/mapbox-navigation-android/issues/3961#issue-804492306)

Reply:
Let's not handle the rotation, just unblock an ability to rotate the screen even if the activity restarts from scratch so that we can see the landscape layout in action.